_data/projects/ramda-adjunct.yml

```
@@ -0,0 +1,20 @@
+---
+name: Ramda Adjunct
+desc: Ramda Adjunct is the most popular and most comprehensive set of functional utilities for use with Ramda, providing a variety of useful, well tested functions with excellent documentation.
+site: https://github.com/char0n/ramda-adjunct
+tags:
+  - ramda
+  - utils
+  - utilities
+  - toolkit
+  - extensions
+  - addons
+  - adjunct
+  - recipes
+  - extras
+  - cookbook
+  - functional
+
+upforgrabs:
+  name: help wanted
+  link: https://github.com/char0n/ramda-adjunct/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

:art: a simpler label

```suggestion
  link: https://github.com/char0n/ramda-adjunct/labels/help%20wanted
```